js_functions/task3.js

```diff
@@ -0,0 +1,11 @@
+function counter(num) {
+    console.log(num)
+    if (num > 1) {
```

Comment:
also good and we swill can make it easier
```js
function counter(num) {
    console.log(num)
    if (num > 1) {
        return counter(num - 1)
    }
    return "Stop function!"
}
```

Reply:
if I  remove 2nd return there is "undefined" in console, how to make it shorter and avoid undefined?
<img width="588" alt="Screenshot 2023-09-12 at 10 08 57" src="https://github.com/TaniaBystrova/aqa-advanced-2/assets/117652009/8bd7c80a-5f2a-4238-9ea2-3c089800f047">

Reply:
I am not sure I understand your question but if I do then I would suggest to do this way
```js
function counter(num) {
    console.log(num)
    if (num > 1) {
        return counter(num - 1)
    }
}

counter(5)
```

Reply:
yes, got it, thank you, that's what I asked, how to remove undefined